src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -59,6 +61,9 @@ public EditCommand parse(String args) throws ParseException {
             editPersonDescriptor.setAddress(ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get()));
         }
         parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).ifPresent(editPersonDescriptor::setTags);
+        if (argMultimap.getValue(PREFIX_PRIORITY).isPresent()) {
```

Comment:
@nigellqx likewise for this part of the code, I think you can try to make use of methods provided by the `Optional` class
You can try something like this.
`argMultimap.getValue(PREFIX_PRIORITY).ifPresent(value -> editPersonDescriptor.setPriority(ParserUtil.parsePriority(value))`

Reply:
For this part, I think I use isPresent() because the above method uses this structure so I think it might be better to follow the convention. What do you think?

Reply:
If that is the case, I think for this part of the code we can just follow the same structure as the previous code segments.